packages/react-native/Libraries/Components/TextInput/AndroidTextInputNativeComponent.js

```diff
@@ -179,6 +179,13 @@ export type NativeProps = $ReadOnly<{|
    */
   numberOfLines?: ?Int32,
 
+  /**
+   * Sets the maximum number of lines for a `TextInput`. Use it with multiline set to
+   * `true` to be able to fill the lines.
+   * @platform android
+   */
+  maximumNumberOfLines?: ?Int32,
```

Comment:
Maybe maxRows? 
I will do my best to apply your suggested changes early next week.

Reply:
I just renamed it in InternalTextInput & flow & TS so that end user can pass `maxNumberOfLines` prop but on the native side we still use `maximumNumberOfLines`. Hope it's fine